jerry-core/vm/vm.c

```diff
@@ -1047,7 +1047,12 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
         }
         case VM_OC_SET_PROPERTY:
         {
+#ifndef CONFIG_DISABLE_ES2015_CLASS
+          const int index = (byte_code_start_p[0] == CBC_EXT_OPCODE) ? -2 : -1;
```

Comment:
Just create a `const int index = - 1` and use the index for accessing the stack top.